File: src/main/java/wang/crick/ddl2plantuml/model/Column.java
package wang.crick.ddl2plantuml.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.io.Serializable;

/**
 * 列
 *
 * @author wangyuheng
 * @date 2018/6/28 11:39
 */
@Data
@AllArgsConstructor
public class Column implements Serializable {

    private String name;
    private String comment;
    private String type;
    private Integer size;
    private String defaultValue;
    private boolean notNull;
}

File: src/main/kotlin/wang/crick/ddl2plantuml/model/Column.kt
package wang.crick.ddl2plantuml.model

import java.io.Serializable

/**
 * 列
 *
 * @author wangyuheng
 * @date 2019-01-20 14:03
 */
class Column(var name: String, var comment: String, var type: String, var size: Int?, var defaultValue: String?, var notNull: Boolean) : Serializable
